css/app.css

```diff
@@ -158,39 +163,11 @@ body {
   color: var(--ptext-color);
 }
 
-.operator:active,
-.number:active {
-  font-size: 15px;
-}
-
-.operator:focus,
-.number:focus {
-  outline: 0;
-}
-
-.number {
-  background: var(--pnumber-color);
-  color: var(--ptext-color);
-  border-radius: 50%;
-}
-
 .operator {
-  background: var(--poperator-color);
+  background-color: #89c0ff;
```

Comment:
if we have defined colors in the theme, why are we dealing with hardcoded colors. Will this change on theme change? Can you please look into this.